pandas/tslib.pyx

```diff
@@ -2388,20 +2418,51 @@ class Timedelta(_Timedelta):
         else:
            return "D"
 
+    def _round(self, freq, rounder):
+
+        cdef int64_t result, unit
+
+        from pandas.tseries.frequencies import to_offset
+        unit = to_offset(freq).nanos
+        result = unit*rounder(self.value/unit)
+        return Timedelta(result,unit='ns')
+
     def round(self, freq):
         """
-        return a new Timedelta rounded to this resolution
+        return a new Timedelta rounded to this resolution.
+
 
         Parameters
         ----------
-        freq : a freq string indicating the rouding resolution
+        freq : a freq string indicating the rounding resolution
+
+        Notes
+        -----
+        For values exactly halfway between rounded decimal values, 
+        Numpy rounds to the nearest even value. Thus the new Timedelta
+        may be lower than the original Timedelta. 
         """
```

Comment:
@jreback  please note the warning I add in the `round` documentation about rounding of `float`. The behaviour is slightly different than the arithmetical definition but it refers to Numpy implementation